make_helpers/defaults.mk

```diff
@@ -128,6 +128,9 @@ TRUSTED_BOARD_BOOT		:= 0
 # Build option to choose whether Trusted firmware uses Coherent memory or not.
 USE_COHERENT_MEM		:= 1
 
+# Use tbbr_oid.h instead of platform_oid.h
+USE_TBBR_DEFS			:= 0
+
```

Comment:
Apparently, the conditional does not work as expected in defaults.mk.

ERROR_DEPRECATED can be overridden much later in platform.mk
but it is always 0 in defaults.mk.  So the code you suggested is equivalent to my code.
So, I assume you suggested me to move this code to a different place.

Or, is it better to do like this?

    USE_TBBR_DEFS              = $(ERROR_DEPRECATED)